README.md

```diff
@@ -564,5 +564,4 @@ This CVE has been reported in systems.text.regularexpressions.dll which is used
 	Snowflake has identified an issue where the driver is globally enforcing TLS 1.2 and certificate revocation checks with the .NET Driver v1.2.1 and earlier versions.  
 	Starting with v2.0.0, the driver will set these locally.  
 	
-	Note that the driver is now targeting .Net framework 4.7.2.  
-	When upgrading to v2.0.0, you might also need to run "Update-Package -reinstall" to update the dependencies.
+  Note that the driver is now targeting .NET 6.0. When upgrading to {release version that targets .NET 6.0}, you might also need to run “Update-Package -reinstall” to update the dependencies.
```

Comment:
> When upgrading to {release version that targets .NET 6.0},

Do the curly braces represent some sort of variable?

If not, could this just be written as "When upgrading to a release version ..." without the curly braces?